axelrod/strategies/cooperator.py

```diff
@@ -37,13 +37,23 @@ class TrickyCooperator(Player):
         'manipulates_state': False
     }
 
-    @staticmethod
-    def strategy(opponent: Player) -> Action:
+    _min_history_required_to_try_trickiness = 3
+    _max_history_depth_for_trickiness = -10
+
+    def strategy(self, opponent: Player) -> Action:
         """Almost always cooperates, but will try to trick the opponent by defecting.
 
-        Defect once in a while in order to get a better payout, when the opponent
-        has not defected in the last ten turns and only cooperated during last 3 turns.
+        Defect once in a while in order to get a better payout.
+        After 3 rounds, if opponent has not defected to a max history depth of 10, Defect.
         """
-        if D not in opponent.history[-10:] and opponent.history[-3:] == [C]*3:
```

Comment:
i had trouble understanding the original intent and wanted to state explicitly what it does.  both ways pass all original tests and tests that i wrote.  was this supposed to be like TrickyDefector, something like `if D in opponent.history and opponent.history[-3:] == [C] * 3` ?